### PR TITLE
Changes for Nextcloud

### PR DESCRIPTION
**Pull request by JSoko**
Changes for Nextcloud

### CODE REVIEW

File: admin_manual/configuration_database/db_conversion.rst [hunk 1]
```diff
@@ -3,17 +3,17 @@ Converting Database Type
...
 
 You can convert a SQLite database to a more performing MySQL, MariaDB or 
-PostgreSQL database with the ownCloud command line tool. SQLite is good for 
-testing and simple single-user ownCloud servers, but it does not scale for 
+PostgreSQL database with the Nextcloud command line tool. SQLite is good for 
+testing and simple single-user Nextcloud servers, but it does not scale for 
 multiple-user production users.
 
-.. note:: ownCloud Enterprise edition does not support SQLite.
+.. note:: Nextcloud Enterprise edition does not support SQLite.
```
Comment (mar1u5) [inline admin_manual/configuration_database/db_conversion.rst]: There is no Netcloud EE 😉 ...so we should remove this part completely.

Comment (JSoko) [inline admin_manual/configuration_database/db_conversion.rst]: OK, I'll remove it and create a new commit

Comment (MorrisJobke) [inline admin_manual/configuration_database/db_conversion.rst]: @JSoko You only fixed the other occurrence. This line here is still there. ;)

Comment (JSoko) [inline admin_manual/configuration_database/db_conversion.rst]: Sorry I'm blind... Changed...

File: admin_manual/installation/nginx_nextcloud_9x.rst [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+      # includeSubDomains; preload;";
+      add_header X-Content-Type-Options nosniff;
+      add_header X-Frame-Options "SAMEORIGIN";
+      add_header X-XSS-Protection "1; mode=block";
+      add_header X-Robots-Tag none;
+      add_header X-Download-Options noopen;
+      add_header X-Permitted-Cross-Domain-Policies none;
+  
+      # Path to the root of your installation
+      root /var/www/owncloud/;
```
Comment (MorrisJobke) [inline admin_manual/installation/nginx_nextcloud_9x.rst]: Nextcloud

File: admin_manual/installation/nginx_nextcloud_9x.rst [hunk 3]
```diff
@@ -0,0 +1,289 @@
+=================================================
+Nginx Configuration for the ownCloud 9.x Branches
+=================================================
+
+The following configuration should be used when ownCloud is placed in the
```
Comment (MorrisJobke) [inline admin_manual/installation/nginx_nextcloud_9x.rst]: Nextcloud

File: admin_manual/installation/linux_installation.rst [hunk 4]
```diff
@@ -2,6 +2,12 @@
 Preferred Linux Installation Method
 ===================================
 
+Nextcloud
+---------
+
+NEXTCLOUD Linux distribution packages are not available at the Moment (June 2016)!
```
Comment (MorrisJobke) [inline admin_manual/installation/linux_installation.rst]: Nextcloud

Comment (MorrisJobke) [inline admin_manual/installation/linux_installation.rst]: And `moment`

File: admin_manual/installation/nginx_nextcloud_9x.rst [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+      }
+  
+      location ~* \.(?:svg|gif|png|html|ttf|woff|ico|jpg|jpeg)$ {
+          try_files $uri /index.php$uri$is_args$args;
+          # Optional: Don't log access to other assets
+          access_log off;
+      }
+  }
+
+ownCloud in a subdir of nginx
```
Comment (MorrisJobke) [inline admin_manual/installation/nginx_nextcloud_9x.rst]: Nextcloud ... also in the remaining part of this file ;)

File: admin_manual/installation/command_line_installation.rst [hunk 6]
```diff
@@ -24,31 +24,31 @@ of running the graphical Installation Wizard::
...
 a PHP fatal error message.
 
 Supported databases are::
 
- - sqlite (SQLite3 - ownCloud Community edition only)
+ - sqlite (SQLite3 - Nextcloud Community edition only)
  - mysql (MySQL/MariaDB)
  - pgsql (PostgreSQL)
- - oci (Oracle - ownCloud Enterprise edition only)
+ - oci (Oracle)
```
Comment (MorrisJobke) [inline admin_manual/installation/command_line_installation.rst]: See https://docs.nextcloud.org/server/9/admin_manual/installation/system_requirements.html#supported-platforms

> Databases: MySQL/MariaDB 5.5+; PostgreSQL; Oracle 11g (currently only possible if you contact us https://nextcloud.com/enterprise as part of a subscription)

File: admin_manual/installation/installation_wizard.rst [hunk 7]
```diff
@@ -57,33 +57,32 @@ and must be owned by your HTTP user (see
...
-:doc:`MySQL/MariaDB <system_requirements>`. Your database and PHP connectors 
-must be installed before you run the Installation Wizard. When you install 
-ownCloud from packages all the necessary dependencies will be satisfied (see 
-:doc:`source_installation` for a detailed listing of required and optional PHP 
-modules). You will need the root database login, or any administrator login that 
-has permissions to create and modify databases, and then enter any name you want 
-for your ownCloud database.
+SQLite is the default database for Nextcloud Server and it is good only for
+testing and lightweight single-user setups without client synchronization.
+Supported databases are MySQL, MariaDB, Oracle 11g , and PostgreSQL, and we
```
Comment (MorrisJobke) [inline admin_manual/installation/installation_wizard.rst]: See https://docs.nextcloud.org/server/9/admin_manual/installation/system_requirements.html#supported-platforms

> Databases: MySQL/MariaDB 5.5+; PostgreSQL; Oracle 11g (currently only possible if you contact us https://nextcloud.com/enterprise as part of a subscription)

**Comment by MorrisJobke** [comment]
@JSoko Thanks a lot. Beside my comments this looks really good :)

File: admin_manual/installation/installation_wizard.rst (hunk 7 above)
Comment (JSoko) [inline admin_manual/installation/installation_wizard.rst]: changed...

File: admin_manual/installation/linux_installation.rst (hunk 4 above)
Comment (JSoko) [inline admin_manual/installation/linux_installation.rst]: changed...

Comment (JSoko) [inline admin_manual/installation/linux_installation.rst]: changed

**Comment by JSoko** [comment]
@MorrisJobke thank you for your work an for your comments... :-))

**Comment by MorrisJobke** [comment]
Makes sense now :) Thanks a lot 👍

**Comment by JSoko** [comment]
Pushed tham reworks more... #37

**Pull request merged**